rsp_vision/analysis/gaussians_calculations.py

```diff
@@ -282,6 +283,9 @@ def get_gaussian_matrix_to_be_plotted(
     matrix_definition : int, optional
```

Comment:
might it make sense to rename `matrix_definition` to `matrix_dimension` ? I was a little confused by what it was before I saw this but maybe it is a well-used term I am unaware of

Reply:
It's made up, I can change it if it's confusing